app/src/main/res/values-land/dimens.xml

```diff
@@ -384,4 +384,10 @@
   <dimen name="profile_rename_activity_profile_rename_input_margin_end">168dp</dimen>
   <dimen name="profile_rename_activity_profile_rename_save_button_margin_end">168dp</dimen>
 
+  <!-- profile_chooser_profile_view -->
```

Comment:
@Rishavgupta12345 here you don't need to use this snake case writing here, you should write it this way <!-- Profile Chooser Profile....--> this way like other headings are written.